### ISSUE TITLE
Implementation of an IncetionpResnet-v2 model for Face Recognition.

### ISSUE THREAD

**Issue by sarobml2000** [akshitagupta15june/Face-X#335]
A InceptionResnet-v2 DNN architecture has to be used for training a Face Recognition model.

**Comment by akshitagupta15june** [comment]
Assigned to u. Please add all your implementation inside this folder https://github.com/akshitagupta15june/Face-X/tree/master/Recognition-Algorithms with a proper readme

**Comment by akshitagupta15june** [comment]
are you still working on this issue

**Comment by sarobml2000** [comment]
Yes, I am. Almost done but trying to get a better validation accuracy.

**Comment by akshitagupta15june** [comment]
are u working on this?

**Comment by sarobml2000** [comment]
Hi @akshitagupta15june, Please assign this and the other issue to someone else. My model was severely overfitting and the classes are way too low due to transfer learning. Thanks.

**Comment by ainy-123** [comment]
hello @akshitagupta15june ,I am GSSoC'21 participant .I want to work on this issue.
Will you please assign this to me ?

**Comment by akshitagupta15june** [comment]
Ok

**Comment by ainy-123** [comment]
> Ok

Thanks for assigning.

**Comment by 1anshu-56** [comment]
I want to work on this issue can you please assign it to me as I have previous experience with OpenCV.

**Comment by abhishek2chikun** [comment]
I want to work on this issue under LGMSOC21. Please assign me.

**Comment by akshitagupta15june** [comment]
as @1anshu-56 is already assigned an issue so I am assigning it to @abhishek2chikun
